torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19456,6 +19456,14 @@ def __init__(
             DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_python_ref_meta',),
         ),
     ),
+    PythonRefInfo(
```

Comment:
relu is an elementwise unary operation, so we should make this (and the relu OpInfo) elementwise unary (that should also test the nan propagation)